src/main/src/main/java/org/geoserver/catalog/impl/WMTSStoreInfoImpl.java

```diff
@@ -31,6 +31,7 @@ public class WMTSStoreInfoImpl extends StoreInfoImpl implements WMTSStoreInfo {
     // Map<String, String> headers;
     private String headerName; // todo: replace with Map<String, String>
     private String headerValue; // todo: replace with Map<String, String>
```

Comment:
This would be the time to replace with `Map<String,String> headers` as indicated in the todo statements above?